app/views/reading_collections/show.html.erb

```diff
@@ -1,6 +1,8 @@
-<%= @collection.name %>
-<% @collection.articles.each do |article| %>
-  <div class="article">
-    <%= article.title %>
-  </div>
-<% end %>
+<% title "All collections" %>
+
+<div class="home">
+        <div class="articleformcontainer" id="collection-articles" data-collection="<%= @collection %>" data-articles="<%= @articles %>">
+        </div>
+</div>
```

Comment:
I see the use for the 2 instance variables from the controller. The BE should have created a serializer for those objects.